curriculum/README.md

```diff
@@ -259,6 +259,10 @@ Coding in Java requires the Java Development Kit and a text editor or IDE. There
 available. Currently most of the TEALS classrooms use Eclipse. Unit 1 includes directions for
 installing Eclipse.
 
+### Practice-It
+
+The TEALS curriculum requires each student to have a copy of the textbook. Many assignments are require students to complete self checks, exercises, and programmings problems at the end of each chapter from the textbook. While practice-it is available from the University of Washington, it is not necessary.  As with all software services, it is the School’s sole decision to use the tool according to the use terms and privacy policies provided by its licensor and it is the School’s responsibility to ensure the tool meets your School’s IT policies.
```

Comment:
The TEALS curriculum requires each student to have a copy of the textbook. Many assignments require students to complete self checks, exercises, and programmings problems at the end of each chapter from the textbook. While Practice-it is available from the University of Washington, it is not necessary.  As with all software services, it is the school’s sole decision to use the tool according to the use terms and privacy policies provided by its licencor and it is the school’s responsibility to ensure the tool meets it's IT policies.